gusto_embedded/README.md

```diff
@@ -116,10 +116,11 @@ Generally, the SDK will work well with most IDEs out of the box. However, when u
 from gusto_embedded import Gusto
 import os
 
-with Gusto(
-    company_access_auth=os.getenv("GUSTO_COMPANY_ACCESS_AUTH", ""),
-) as gusto:
+auth_token = os.getenv("GUSTO_COMPANY_ACCESS_AUTH", None)
```

Comment:
Some of @satanas's feedback – shorter lines, more validation, and use `None` over `""`